mautic/mautic
# [UI] Accordions.less

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #13163 



## Description

Dependency:
- #13630

This PR adds an accordion component. It's based on the Bootstrap JS logic (that's why there is a .panel in the code), but with unique style.


https://github.com/mautic/mautic/assets/116097999/61191d92-0fd7-4d15-bd8b-9c96b580e738








---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. See if everything is the same across the interface, no changes since it's a technical addition

But PRs where you can interact with it:
- https://github.com/mautic/mautic/pull/13939
- https://github.com/mautic/mautic/pull/13912
- https://github.com/mautic/mautic/pull/13942
